### ISSUE TITLE
Reading of Values 

### ISSUE THREAD

**Issue by schanzer** [ds26gte/code.pyret.org#91]
- [x] 1/2 should read as "one over two, a rational number".  Right now it's missing the "over"
- [x] ~99 should read as "~99, a rough number"

**Comment by ds26gte** [comment]
This issue has been solved and some.

- Recite rational numbers according to their format: decimal or fractional. If decimal, and with repeating digits, the latter are also recited appropriately. If fractional, with "over".
- Roughnums are recited with a trailing "roughly".
- Interaction history array stores indices into the REPL DOM for non-errors. This allows history items to be recited with the current format (which is user-toggleable at any time).
